code/modules/power/lighting.dm

```diff
@@ -198,6 +206,9 @@
 /obj/machinery/light/spot
 	name = "spotlight"
 	desc = "A more robust socket for light tubes that demand more power."
+	icon = 'infinity/icons/obj/lighting_inf.dmi' // inf|
+	base_state = "spot" // inf
+	item_state = "spot-construct-item" //inf
```

Comment:
```suggestion
	//[INF]
	icon = 'infinity/icons/obj/lighting_inf.dmi'
	base_state = "spot"
	item_state = "spot-construct-item"
	//[/INF]
```
https://github.com/infinitystation/Baystation12/blob/master/.github/CODE_GUIDELINES.md#%D1%80%D0%B5%D0%B4%D0%B0%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5-%D0%BE%D1%80%D0%B8%D0%B3%D0%B8%D0%BD%D0%B0%D0%BB%D1%8C%D0%BD%D0%BE%D0%B3%D0%BE-%D0%BA%D0%BE%D0%B4%D0%B0